package.json

```diff
@@ -591,6 +591,10 @@
                             "OpenInCurrentWindow"
                         ],
                         "description": "%azFunc.projectOpenBehaviorDescription%"
+                    },
+                    "azureFunctions.python.venvRelativePath": {
```

Comment:
Can we change this to `venvName` since that's what it is now?

Reply:
Hmm, it could still be a path.  For example, if I link it to a venv outside of my project, the setting will look like:

`..\\pythonProject\\func_env`

Maybe we should talk about whether or not we should support existing venvs.